content/src/main/content/jcr_root/apps/acs-commons/components/utilities/system-notifications/notification/notification.jsp

```diff
@@ -50,13 +50,15 @@
     String message = properties.get("jcr:description", String.class);
     message = systemNotifications.getMessage(message, onTimeFormatted, offTimeFormatted);
     pageContext.setAttribute("message", message);
-    
-    pageContext.setAttribute("dismissible", properties.get("dismissible", true));
+
+
+    boolean dismissible = properties.get("dismissible", true);
```

Comment:
Shouldn't this be converted to HTL as well?

Reply:
That's something I'll do tomorrow :-)

Reply:
Was it done?  If so, happy to merge it once it is.